runtime/regionx/src/adapter.rs

```diff
@@ -0,0 +1,323 @@
+// TODO: remove this file
```

Comment:
This has been temporarily added here because the NonFungibleAdapter is in release 1.7.0 or later, but we cannot update to that version now because of ISMP.